src/components/comment/view/commentView.js

```diff
@@ -56,115 +57,121 @@ class CommentView extends PureComponent {
-                  name="people"
-                />
-                <Text style={styles.voteCountText}>{voteCount}</Text>
-                <IconButton
-                  size={18}
-                  iconStyle={styles.leftIcon}
-                  style={styles.leftButton}
-                  name="reply"
-                  onPress={() => handleOnReplyPress && handleOnReplyPress(comment)}
-                  iconType="MaterialIcons"
-                />
-                {currentAccountUsername === comment.author && (
-                  <Fragment>
-                    <IconButton
-                      size={18}
-                      iconStyle={styles.leftIcon}
-                      style={styles.leftButton}
-                      name="create"
-                      onPress={() => handleOnEditPress && handleOnEditPress(comment)}
-                      iconType="MaterialIcons"
-                    />
-                    {!comment.children && !voteCount && (
-                      <Fragment>
-                        <IconButton
-                          size={18}
-                          iconStyle={styles.leftIcon}
-                          style={styles.leftButton}
-                          name="delete-forever"
-                          onPress={() => this.ActionSheet.show()}
-                          iconType="MaterialIcons"
-                        />
-                        <ActionSheet
-                          ref={o => (this.ActionSheet = o)}
-                          options={[
-                            intl.formatMessage({ id: 'alert.delete' }),
-                            intl.formatMessage({ id: 'alert.cancel' }),
-                          ]}
-                          title={intl.formatMessage({ id: 'alert.delete' })}
-                          destructiveButtonIndex={0}
-                          cancelButtonIndex={1}
-                          onPress={index => {
-                            index === 0 ? handleDeleteComment(comment.permlink) : null;
-                          }}
-                        />
-                      </Fragment>
-                    )}
-                  </Fragment>
-                )}
-              </Fragment>
-            )}
-            {isShowMoreButton && (
-              <View style={styles.rightButtonWrapper}>
-                <TextWithIcon
-                  wrapperStyle={styles.rightButton}
-                  iconName={isShowSubComments ? 'keyboard-arrow-up' : 'keyboard-arrow-down'}
-                  textStyle={!isPressedShowButton && styles.moreText}
-                  iconType="MaterialIcons"
-                  isClickable
-                  iconStyle={styles.iconStyle}
-                  iconSize={16}
-                  onPress={() => this._showSubCommentsToggle()}
-                  text={!isPressedShowButton ? `${comment.children} more replies` : ''}
-                />
-              </View>
+      <TouchableWithoutFeedback onLongPress={handleOnLongPress}>
+        <View>
```

Comment:
remove

Reply:
did you changed latest view tag but first one is still there is it correct or I am seeing wrong tag?

Reply:
we should user just 1 tag inside of TouchableWithoutFeedback, this is why I didnt remove view tag